node/src/components/event_stream_server/event.rs

```diff
@@ -38,6 +42,11 @@ impl Display for Event {
                 public_key, timestamp, era_id,
             ),
             Event::FinalitySignature(fs) => write!(formatter, "finality signature {}", fs),
+            Event::Step { era_id, effect } => write!(
+                formatter,
+                "Execution effect: {:?} committed in era: {}",
```

Comment:
`Display` implementations generally shouldn't use `Debug` output internally.  Maybe we should remove this `{:?}` and ticket an update for EE?

Also, `Display` impls should generally be lowercase, and the output doesn't mention "step" anywhere, so I'd change this to something like `"step completed for {}", era_id`.

Reply:
Applied suggestion